consumer_test.go

```diff
@@ -223,3 +233,9 @@ func consumerTest(t *testing.T, cb func(c *Config)) {
 		t.Fatal("failed message not done")
 	}
 }
+
+func port() string {
```

Comment:
should we listen on `:0`, read the port, close, then return that assumed open port?